From 7ac57c5c7041b4b3482b4d3dddb10daf7e87321e Mon Sep 17 00:00:00 2001
From: lvqiang214 <lvqiang1@huawei.com>
Date: Fri, 7 Jul 2023 16:19:34 +0800
Subject: [PATCH] add intelligent voice core

Signed-off-by: lvqiang214 <lvqiang1@huawei.com>
---
 include/codec_config/syscap_define.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/codec_config/syscap_define.h b/include/codec_config/syscap_define.h
index 54cc6ad..39d46f2 100755
--- a/include/codec_config/syscap_define.h
+++ b/include/codec_config/syscap_define.h
@@ -299,6 +299,7 @@ typedef enum SystemCapabilityNum {
     CLOUD_ADS,
     MULTIMEDIA_AUDIO_PLAYBACKCAPTURE,
     MULTIMEDIA_MEDIA_AVSCREENCAPTURE,
+    AI_INTELLIGENTVOICE_CORE,
     // Add before here
     SYSCAP_BASIC_END = 500,
 } SyscapNum;
@@ -580,6 +581,7 @@ const static SyscapWithNum g_arraySyscap[] = {
     {"SystemCapability.Cloud.Ads", CLOUD_ADS},
     {"SystemCapability.Multimedia.Audio.PlaybackCapture", MULTIMEDIA_AUDIO_PLAYBACKCAPTURE},
     {"SystemCapability.Multimedia.Media.AVScreenCapture", MULTIMEDIA_MEDIA_AVSCREENCAPTURE},
+    {"SystemCapability.AI.IntelligentVoice.Core", AI_INTELLIGENTVOICE_CORE},
 };
 
 #ifdef __cplusplus
-- 
2.34.1

